lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -162,14 +162,17 @@ function findInheritedCSSRule(inheritedEntries = []) {
  * @param {LH.Crdp.CSS.GetMatchedStylesForNodeResponse} matched CSS rules
  * @returns {NodeFontData['cssRule']|undefined}
  */
-function getEffectiveFontRule({inlineStyle, matchedCSSRules, inherited}) {
+function getEffectiveFontRule({attributesStyle, inlineStyle, matchedCSSRules, inherited}) {
   // Inline styles have highest priority
   if (hasFontSizeDeclaration(inlineStyle)) return {type: 'Inline', ...inlineStyle};
 
   // Rules directly referencing the node come next
   const matchedRule = findMostSpecificMatchedCSSRule(matchedCSSRules);
   if (matchedRule) return matchedRule;
 
+  // Then comes attributes styles (<font size="1">)
```

Comment:
I'm a little confused, doesn't your description say that `attributesStyle` is just a set of undefined properties?

Reply:
It'll have style info, but the CSSStyle won't have a stylesheet or a range :)

https://chromedevtools.github.io/devtools-protocol/tot/CSS#method-getMatchedStylesForNode